script/clone_job.pl

```diff
@@ -113,7 +113,7 @@ =head1 SYNOPSIS
 use JSON;
 
 use FindBin;
-use lib "$FindBin::Bin/../lib";
+BEGIN { unshift @INC, "$FindBin::RealBin/../lib" }
 use OpenQA::Client;
```

Comment:
sure, the use case makes perfectly sense. IMO  `use lib` is less clunky as a syntax to achieve the same effect wrt unshifting `@INC`, makes it explicit and more error-prone: aka an invitation to play with the BEGIN block :)